

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Ras regulation by RasGAPs &mdash; Ras model 0.1 documentation</title>
  

  
  

  

  
  
    

  

  
  
    <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  

  

  
    <link rel="top" title="Ras model 0.1 documentation" href="../../index.html"/>
        <link rel="up" title="Ras" href="index.html"/>
        <link rel="next" title="Ras (minimal)" href="ras_minimal.html"/>
        <link rel="prev" title="Ras regulation by GEFs" href="ras_gefs.html"/> 

  
  <script src="../../_static/js/modernizr.min.js"></script>

</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search">
          

          
            <a href="../../index.html" class="icon icon-home"> Ras model
          

          
          </a>

          
            
            
              <div class="version">
                0.1
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
                <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../../intro.html">Overview</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="../index.html">The RAS Executable Model</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="../egfr/index.html">EGFR</a></li>
<li class="toctree-l2 current"><a class="reference internal" href="index.html">Ras</a><ul class="current">
<li class="toctree-l3"><a class="reference internal" href="ras_gtpase.html">Ras proteins are GTPases</a></li>
<li class="toctree-l3"><a class="reference internal" href="ras_gefs.html">Ras regulation by GEFs</a></li>
<li class="toctree-l3 current"><a class="current reference internal" href="">Ras regulation by RasGAPs</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#rasgaps-for-the-ras-subfamily">RasGAPs for the Ras subfamily</a></li>
<li class="toctree-l4"><a class="reference internal" href="#mechanism-of-accelerating-hydrolysis">Mechanism of accelerating hydrolysis</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="ras_minimal.html">Ras (minimal)</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../raf/index.html">Raf</a></li>
<li class="toctree-l2"><a class="reference internal" href="../mapk/index.html">MAPK</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../scenarios/index.html">Model scenarios</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../experiments/index.html">Model analysis</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../references.html">References</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="../../index.html">Ras model</a>
      </nav>


      
      <div class="wy-nav-content">
        <div class="rst-content">
          





<div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="../../index.html">Docs</a> &raquo;</li>
      
          <li><a href="../index.html">The RAS Executable Model (REM)</a> &raquo;</li>
      
          <li><a href="index.html">Ras</a> &raquo;</li>
      
    <li>Ras regulation by RasGAPs</li>
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="../../_sources/components/ras/ras_gaps.txt" rel="nofollow"> View page source</a>
          
        
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="ras-regulation-by-rasgaps">
<h1>Ras regulation by RasGAPs<a class="headerlink" href="#ras-regulation-by-rasgaps" title="Permalink to this headline">¶</a></h1>
<p>GTP hydrolysis by Ras is slow but is accelerated by RasGAPs.</p>
<blockquote>
<div><p><a class="reference internal" href="../../references.html#pmid9247124" id="id1">[PMID9247124]</a>: The GTP-binding proteins return to the inactive state by
virtue of the GTPase reaction, which is usually very slow but can be
accelerated by the action of GAPs, in the case of the Ras/Ras-GAPs and
Ran/Ran-GAP interactions by several orders of magnitude <a class="reference internal" href="../../references.html#pmid1569940" id="id2">[PMID1569940]</a>
<a class="reference internal" href="../../references.html#pmid8262937" id="id3">[PMID8262937]</a> <a class="reference internal" href="../../references.html#pmid7548002" id="id4">[PMID7548002]</a>.</p>
</div></blockquote>
<div class="section" id="rasgaps-for-the-ras-subfamily">
<h2>RasGAPs for the Ras subfamily<a class="headerlink" href="#rasgaps-for-the-ras-subfamily" title="Permalink to this headline">¶</a></h2>
<p>There are several GAPs for the Ras subfamily.</p>
<blockquote>
<div><p><a class="reference internal" href="../../references.html#pmid9247124" id="id5">[PMID9247124]</a>: Five mammalian GAPs for Ras have been described. The first,
p120GAP, is the prototype of this class of proteins and was the first one
to be isolated (20, 21 and 22). Apart from being a regulator of Ras, its
N-terminal domain contains a number of signalling modules such as SH2, SH3,
PH, Calb domains and is believed to be a signal transduction molecule that
may act independently of Ras (23 and 24).</p>
</div></blockquote>
<div class="section" id="rasa1">
<h3>RASA1<a class="headerlink" href="#rasa1" title="Permalink to this headline">¶</a></h3>
<p>The domain information for RASA1 was taken from <a class="reference external" href="http://www.uniprot.org/uniprot/P20936">Uniprot (ID: P20936)</a>.</p>
<div class="highlight-python"><div class="highlight"><pre><span class="k">def</span> <span class="nf">rasgap_monomers</span><span class="p">():</span>

    <span class="c">#  p120GAP = RASA1.</span>
    <span class="c"># CaLB (calcium lipid binding domain) is also known as a C2 domain.</span>
    <span class="n">Monomer</span><span class="p">(</span><span class="s">&#39;RASA1&#39;</span><span class="p">,</span> <span class="p">[</span><span class="s">&#39;SH2_1&#39;</span><span class="p">,</span> <span class="s">&#39;SH3&#39;</span><span class="p">,</span> <span class="s">&#39;SH2_2&#39;</span><span class="p">,</span> <span class="s">&#39;PH&#39;</span><span class="p">,</span> <span class="s">&#39;C2&#39;</span><span class="p">,</span> <span class="s">&#39;rasgap&#39;</span><span class="p">])</span>
</pre></div>
</div>
</div>
<div class="section" id="nf1">
<h3>NF1<a class="headerlink" href="#nf1" title="Permalink to this headline">¶</a></h3>
<blockquote>
<div><p><a class="reference internal" href="../../references.html#pmid9247124" id="id6">[PMID9247124]</a>: The second Ras-specific GAP is neurofibromin (NF1), which
is the product of the neurofibromatosis gene [PMID25] and has also been
shown to stimulate the GTPase of Ras 26, 27 and 28.  This gene has been
found to be frequently mutated in patients with the disease
neurofibromatosis type I 29, 30 and 31 but also, albeit less frequently, in
solid tumors [PMID32].</p>
</div></blockquote>
<p><a class="reference external" href="http://www.uniprot.org/uniprot/P21359">Uniprot (ID: P21359)</a>.</p>
<div class="highlight-python"><div class="highlight"><pre>    <span class="n">Monomer</span><span class="p">(</span><span class="s">&#39;NF1&#39;</span><span class="p">,</span> <span class="p">[</span><span class="s">&#39;rasgap&#39;</span><span class="p">,</span> <span class="s">&#39;CRALTRIO&#39;</span><span class="p">])</span>
</pre></div>
</div>
</div>
<div class="section" id="rasa2">
<h3>RASA2<a class="headerlink" href="#rasa2" title="Permalink to this headline">¶</a></h3>
<blockquote>
<div><p><a class="reference internal" href="../../references.html#pmid9247124" id="id7">[PMID9247124]</a>: GAP1m, a mammalian homologue of the Drosophila GAP1 gene
[PMID33], has been described, and a close homologue GAPIII [PMID34], both
of which contain, in addition to the GRD (GAP-related domain), C2 domains
and a PH domain.</p>
</div></blockquote>
<p><a class="reference external" href="http://www.uniprot.org/uniprot/Q15283">Uniprot (ID: Q15283)</a>.</p>
<div class="highlight-python"><div class="highlight"><pre>    <span class="c"># GAP1m = RASA2</span>
    <span class="n">Monomer</span><span class="p">(</span><span class="s">&#39;RASA2&#39;</span><span class="p">,</span> <span class="p">[</span><span class="s">&#39;C2_1&#39;</span><span class="p">,</span> <span class="s">&#39;C2_2&#39;</span><span class="p">,</span> <span class="s">&#39;rasgap&#39;</span><span class="p">,</span> <span class="s">&#39;PH&#39;</span><span class="p">,</span> <span class="s">&#39;BTK&#39;</span><span class="p">])</span>
</pre></div>
</div>
</div>
<div class="section" id="rasa3">
<h3>RASA3<a class="headerlink" href="#rasa3" title="Permalink to this headline">¶</a></h3>
<p>NOTE: As it turned out GAPIII and GAP1IP4BP are the same protein.</p>
<blockquote>
<div><p><a class="reference internal" href="../../references.html#pmid9247124" id="id8">[PMID9247124]</a>: Recently an inositol-4-phosphate (IP4) binding protein
GAP1IP4BP has been purified, cloned, and found to contain a Ras-GAP
catalytic domain.  In contrast to the other GAP mentioned, which are
specific for Ras, GAP1IP4BP stimulates the GTPase of both Ras and Rap
[PMID35].</p>
</div></blockquote>
<p><a class="reference external" href="http://www.uniprot.org/uniprot/Q14644">Uniprot (ID: Q14644)</a>.</p>
<div class="highlight-python"><div class="highlight"><pre>    <span class="c"># GAPIII = GAP1IP4BP = RASA3</span>
    <span class="n">Monomer</span><span class="p">(</span><span class="s">&#39;RASA3&#39;</span><span class="p">,</span> <span class="p">[</span><span class="s">&#39;C2_1&#39;</span><span class="p">,</span> <span class="s">&#39;C2_2&#39;</span><span class="p">,</span> <span class="s">&#39;rasgap&#39;</span><span class="p">,</span> <span class="s">&#39;PH&#39;</span><span class="p">,</span> <span class="s">&#39;BTK&#39;</span><span class="p">])</span>
</pre></div>
</div>
</div>
</div>
<div class="section" id="mechanism-of-accelerating-hydrolysis">
<h2>Mechanism of accelerating hydrolysis<a class="headerlink" href="#mechanism-of-accelerating-hydrolysis" title="Permalink to this headline">¶</a></h2>
<blockquote>
<div><p><a href="#id11"><span class="problematic" id="id12"><span id="id9"></span>[9247124]_</span></a>: Under saturating conditions of pl20GAP, the <span class="math">\(k_{cat}\)</span> of
the GTPase reaction of Ras is 19 <span class="math">\(s^{-1}\)</span>, which means that the
reaction is stimulated more than <span class="math">\(10^5\)</span>-fold, with a KD of 9.7 uM for
the GAP/Ras-GTP interaction <a href="#id13"><span class="problematic" id="id14"><span id="id10"></span>[1569940]_</span></a>.</p>
</div></blockquote>
<p>Because the co-crystal structure of Ras with RasGAP shows the occlusion of the
GTP binding pocket by RasGAP, we model this by asserting that the RasGAP binds
at the Switch 1/Switch 2 region on Ras. This has the effect of preventing the
binding of downstream effectors such as Raf.</p>
<p>Further, for the time being we assume that RasGAPs bind only to the GTP-bound
state, in which s1s2 are &#8216;closed&#8217;, i.e., once the nucleotide is in the tightly
bound state; and that a RasGAP and a RasGEF cannot simultaneously be bound to
the same Ras molecule.</p>
<p>These assumptions yield the following mechanism:</p>
<div class="highlight-python"><div class="highlight"><pre><span class="k">def</span> <span class="nf">rasgap_mediated_hydrolysis</span><span class="p">(</span><span class="n">model</span><span class="p">,</span> <span class="n">ras</span><span class="p">,</span> <span class="n">rasgap</span><span class="p">,</span> <span class="n">kf</span><span class="p">,</span> <span class="n">kr</span><span class="p">,</span> <span class="n">kcat</span><span class="p">):</span>
    <span class="n">GTP</span> <span class="o">=</span> <span class="n">model</span><span class="o">.</span><span class="n">monomers</span><span class="p">[</span><span class="s">&#39;GTP&#39;</span><span class="p">]</span>
    <span class="n">GDP</span> <span class="o">=</span> <span class="n">model</span><span class="o">.</span><span class="n">monomers</span><span class="p">[</span><span class="s">&#39;GDP&#39;</span><span class="p">]</span>
    <span class="n">Pi</span> <span class="o">=</span> <span class="n">model</span><span class="o">.</span><span class="n">monomers</span><span class="p">[</span><span class="s">&#39;Pi&#39;</span><span class="p">]</span>
    <span class="c"># RasGAP, with an unbound RasGAP domain, binds to GTP-bound Ras with</span>
    <span class="c"># its Switch 1/Switch 2 region in the &#39;closed&#39; state, with no RasGEF</span>
    <span class="c"># bound:</span>
    <span class="n">ras_gtp</span> <span class="o">=</span> <span class="n">ras</span><span class="p">(</span><span class="n">s1s2</span><span class="o">=</span><span class="s">&#39;closed&#39;</span><span class="p">,</span> <span class="n">gef</span><span class="o">=</span><span class="bp">None</span><span class="p">,</span> <span class="n">gtp</span><span class="o">=</span><span class="mi">2</span><span class="p">)</span> <span class="o">%</span> <span class="n">GTP</span><span class="p">(</span><span class="n">p</span><span class="o">=</span><span class="mi">2</span><span class="p">)</span>
    <span class="n">ras_gdp</span> <span class="o">=</span> <span class="n">ras</span><span class="p">(</span><span class="n">s1s2</span><span class="o">=</span><span class="s">&#39;closed&#39;</span><span class="p">,</span> <span class="n">gef</span><span class="o">=</span><span class="bp">None</span><span class="p">,</span> <span class="n">gtp</span><span class="o">=</span><span class="mi">2</span><span class="p">)</span> <span class="o">%</span> <span class="n">GDP</span><span class="p">(</span><span class="n">p</span><span class="o">=</span><span class="mi">2</span><span class="p">)</span>
    <span class="n">bind</span><span class="p">(</span><span class="n">rasgap</span><span class="p">,</span> <span class="s">&#39;rasgap&#39;</span><span class="p">,</span> <span class="n">ras_gtp</span><span class="p">,</span> <span class="s">&#39;gap&#39;</span><span class="p">,</span> <span class="p">[</span><span class="n">kf</span><span class="p">,</span> <span class="n">kr</span><span class="p">])</span>
    <span class="c"># Because a GDP is being created on the right-hand side of this rule,</span>
    <span class="c"># it must be fully specified. This means that we need to handle</span>
    <span class="c"># the case when the GTP/GDP is either labeled or unlabeled.</span>
    <span class="k">for</span> <span class="n">label</span> <span class="ow">in</span> <span class="p">[</span><span class="s">&#39;y&#39;</span><span class="p">,</span> <span class="s">&#39;n&#39;</span><span class="p">]:</span>
        <span class="n">_macro_rule</span><span class="p">(</span><span class="s">&#39;catalyze&#39;</span><span class="p">,</span>
                    <span class="n">rasgap</span><span class="p">(</span><span class="n">rasgap</span><span class="o">=</span><span class="mi">1</span><span class="p">)</span> <span class="o">%</span> <span class="n">ras_gtp</span><span class="p">(</span><span class="n">gap</span><span class="o">=</span><span class="mi">1</span><span class="p">,</span> <span class="n">label</span><span class="o">=</span><span class="n">label</span><span class="p">)</span> <span class="o">&gt;&gt;</span>
                    <span class="n">rasgap</span><span class="p">(</span><span class="n">rasgap</span><span class="o">=</span><span class="bp">None</span><span class="p">)</span> <span class="o">+</span>
                        <span class="n">ras_gdp</span><span class="p">(</span><span class="n">gap</span><span class="o">=</span><span class="bp">None</span><span class="p">,</span> <span class="n">label</span><span class="o">=</span><span class="n">label</span><span class="p">)</span> <span class="o">+</span> <span class="n">Pi</span><span class="p">(),</span>
                    <span class="p">[</span><span class="n">kcat</span><span class="p">],</span> <span class="p">[</span><span class="s">&#39;kcat&#39;</span><span class="p">])</span>
</pre></div>
</div>
<p>For the time being, we include only the hydrolysis reaction for KRAS,
mediated by RASA1 (P120GAP):</p>
<div class="highlight-python"><div class="highlight"><pre><span class="k">def</span> <span class="nf">kras_rasgaps</span><span class="p">(</span><span class="n">model</span><span class="p">):</span>
    <span class="n">KRAS</span> <span class="o">=</span> <span class="n">model</span><span class="o">.</span><span class="n">monomers</span><span class="p">[</span><span class="s">&#39;KRAS&#39;</span><span class="p">]</span>
    <span class="n">RASA1</span> <span class="o">=</span> <span class="n">model</span><span class="o">.</span><span class="n">monomers</span><span class="p">[</span><span class="s">&#39;RASA1&#39;</span><span class="p">]</span>

    <span class="c"># Diffusion-limited KRAS/RasGAP association:</span>
    <span class="n">kf</span> <span class="o">=</span> <span class="mf">1e-2</span>
    <span class="c"># Calculate dissociation rate based on roughly 10 uM KD reported above</span>
    <span class="n">kr</span> <span class="o">=</span> <span class="mf">100.</span> <span class="c"># KD = 100 / 0.01 = 10,000 nM = 10 uM</span>

    <span class="n">hydrolysis_rates</span> <span class="o">=</span> <span class="p">{</span><span class="s">&#39;WT&#39;</span><span class="p">:</span> <span class="mf">4300e-5</span><span class="p">,</span>
                        <span class="s">&#39;G12A&#39;</span><span class="p">:</span> <span class="mf">32e-5</span><span class="p">,</span>
                        <span class="s">&#39;G12C&#39;</span><span class="p">:</span> <span class="mf">20e-5</span><span class="p">,</span>
                        <span class="s">&#39;G12D&#39;</span><span class="p">:</span> <span class="mf">89e-5</span><span class="p">,</span>
                        <span class="s">&#39;G12R&#39;</span><span class="p">:</span> <span class="mf">20e-5</span><span class="p">,</span>
                        <span class="s">&#39;G12V&#39;</span><span class="p">:</span> <span class="mf">24e-5</span><span class="p">,</span>
                        <span class="s">&#39;G13D&#39;</span><span class="p">:</span> <span class="mf">20e-5</span><span class="p">,</span>
                        <span class="s">&#39;Q61L&#39;</span><span class="p">:</span> <span class="mf">12e-5</span><span class="p">,</span>
                        <span class="s">&#39;Q61H&#39;</span><span class="p">:</span> <span class="mf">5e-5</span><span class="p">}</span>
    <span class="c"># Iterate over all of the mutants that we&#39;re considering</span>
    <span class="k">for</span> <span class="n">mutant</span> <span class="ow">in</span> <span class="n">KRAS</span><span class="o">.</span><span class="n">site_states</span><span class="p">[</span><span class="s">&#39;mutant&#39;</span><span class="p">]:</span>
        <span class="n">kras</span> <span class="o">=</span> <span class="n">KRAS</span><span class="p">(</span><span class="n">mutant</span><span class="o">=</span><span class="n">mutant</span><span class="p">)</span>
        <span class="k">if</span> <span class="n">mutant</span> <span class="ow">in</span> <span class="n">hydrolysis_rates</span><span class="p">:</span>
            <span class="n">mutant_kcat</span> <span class="o">=</span> <span class="n">hydrolysis_rates</span><span class="p">[</span><span class="n">mutant</span><span class="p">]</span>
        <span class="k">else</span><span class="p">:</span>
            <span class="n">mutant_kcat</span> <span class="o">=</span> <span class="n">hydrolysis_rates</span><span class="p">[</span><span class="s">&#39;WT&#39;</span><span class="p">]</span>
        <span class="n">rasgap_mediated_hydrolysis</span><span class="p">(</span><span class="n">model</span><span class="p">,</span> <span class="n">kras</span><span class="p">,</span> <span class="n">RASA1</span><span class="p">,</span> <span class="n">kf</span><span class="p">,</span> <span class="n">kr</span><span class="p">,</span> <span class="n">mutant_kcat</span><span class="p">)</span>
</pre></div>
</div>
</div>
</div>


           </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="ras_minimal.html" class="btn btn-neutral float-right" title="Ras (minimal)" accesskey="n">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="ras_gefs.html" class="btn btn-neutral" title="Ras regulation by GEFs" accesskey="p"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2015, Sorger Lab.

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  

    <script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'../../',
            VERSION:'0.1',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true
        };
    </script>
      <script type="text/javascript" src="../../_static/jquery.js"></script>
      <script type="text/javascript" src="../../_static/underscore.js"></script>
      <script type="text/javascript" src="../../_static/doctools.js"></script>
      <script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

  

  
  
    <script type="text/javascript" src="../../_static/js/theme.js"></script>
  

  
  
  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.StickyNav.enable();
      });
  </script>
   

</body>
</html>