

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Ras regulation by GEFs &mdash; Ras model 0.1 documentation</title>
  

  
  

  

  
  
    

  

  
  
    <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  

  

  
    <link rel="top" title="Ras model 0.1 documentation" href="../../index.html"/>
        <link rel="up" title="Ras" href="index.html"/>
        <link rel="next" title="Ras regulation by RasGAPs" href="ras_gaps.html"/>
        <link rel="prev" title="Ras proteins are GTPases" href="ras_gtpase.html"/> 

  
  <script src="../../_static/js/modernizr.min.js"></script>

</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search">
          

          
            <a href="../../home.html" class="icon icon-home"> Ras model
          

          
          </a>

          
            
            
              <div class="version">
                0.1
              </div>
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
                <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="../../intro.html">Ras Executable Model Overview</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="../index.html">The RAS Executable Model (REM)</a><ul class="current">
<li class="toctree-l2 current"><a class="reference internal" href="index.html">Ras</a><ul class="current">
<li class="toctree-l3"><a class="reference internal" href="ras_gtpase.html">Ras proteins are GTPases</a></li>
<li class="toctree-l3 current"><a class="current reference internal" href="">Ras regulation by GEFs</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#identities-of-ras-specific-gefs">Identities of Ras-specific GEFs</a></li>
<li class="toctree-l4"><a class="reference internal" href="#mechanism-of-gefs">Mechanism of GEFs</a></li>
<li class="toctree-l4"><a class="reference internal" href="#rates-of-gef-activation">Rates of GEF activation</a></li>
<li class="toctree-l4"><a class="reference internal" href="#specificity-of-rasgrf1-for-ras-isoforms">Specificity of RASGRF1 for Ras isoforms</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="ras_gaps.html">Ras regulation by RasGAPs</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../raf/index.html">Raf</a></li>
<li class="toctree-l2"><a class="reference internal" href="../initial_conditions.html">Default initial conditions</a></li>
<li class="toctree-l2"><a class="reference internal" href="../observables.html">Observables</a></li>
<li class="toctree-l2"><a class="reference internal" href="../references.html">References</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../../assembled/index.html">Assembled model</a></li>
<li class="toctree-l1"><a class="reference internal" href="../../results/index.html">Model outputs</a></li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="../../home.html">Ras model</a>
      </nav>


      
      <div class="wy-nav-content">
        <div class="rst-content">
          





<div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="../../home.html">Docs</a> &raquo;</li>
      
          <li><a href="../index.html">The RAS Executable Model (REM)</a> &raquo;</li>
      
          <li><a href="index.html">Ras</a> &raquo;</li>
      
    <li>Ras regulation by GEFs</li>
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="../../_sources/doc_model/ras/ras_gefs.txt" rel="nofollow"> View page source</a>
          
        
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="ras-regulation-by-gefs">
<span id="ras-gefs"></span><h1>Ras regulation by GEFs<a class="headerlink" href="#ras-regulation-by-gefs" title="Permalink to this headline">¶</a></h1>
<div class="section" id="identities-of-ras-specific-gefs">
<h2>Identities of Ras-specific GEFs<a class="headerlink" href="#identities-of-ras-specific-gefs" title="Permalink to this headline">¶</a></h2>
<p>There are several guanosine exchange factors (GEFs) that are specific to Ras
subfamily proteins.</p>
<blockquote>
<div><a class="reference internal" href="../references.html#pmid9585556" id="id1">[PMID9585556]</a>: Several genes have been isolated from different organisms
encoding proteins that have a GEF activity specific for Ras (for which we
use the general name RasGEFs throughout this paper): SOS1 and SOS2
(<a class="reference internal" href="../references.html#pmid9585556-1" id="id2">[PMID9585556_1]</a> <a class="reference internal" href="../references.html#pmid9585556-2" id="id3">[PMID9585556_2]</a> <a class="reference internal" href="../references.html#pmid9585556-3" id="id4">[PMID9585556_3]</a> <a class="reference internal" href="../references.html#pmid9585556-4" id="id5">[PMID9585556_4]</a>) ;
Cdc25Mm, also called RasGrf (<a class="reference internal" href="../references.html#pmid9585556-5" id="id6">[PMID9585556_5]</a> <a class="reference internal" href="../references.html#pmid9585556-6" id="id7">[PMID9585556_6]</a>
<a class="reference internal" href="../references.html#pmid9585556-7" id="id8">[PMID9585556_7]</a>); and mRas-GRF2 (<a class="reference internal" href="../references.html#pmid9585556-8" id="id9">[PMID9585556_8]</a>).</div></blockquote>
<p>RasGEFs contain a specific domain responsible for activating Ras proteins.</p>
<blockquote>
<div><a class="reference internal" href="../references.html#pmid9585556" id="id10">[PMID9585556]</a>: The RasGEFs are proteins of considerable length, 120 - 160
kDa, and contain several regions which are generally accepted to represent
structural domains (12). A region of 200 - 300 amino acids, the RasGEF
domain, is shared by all GEFs which act on members of the Ras subfamily,
and their activity is specific toward either Ras, Ral, or Rap. The fact
that truncated versions of various lengths, containing this RasGEF domain,
have been shown to be active RasGEFs in vivo and in vitro (4 , 13 - 16)
confirms that this region indeed represents the Ras-specific guanine
nucleotide exchange domain.</div></blockquote>
<div class="highlight-python"><div class="highlight"><pre><span class="c"># Declare a list of RasGEFs along with their site structure.</span>
<span class="c"># The names in the list below are HGNC standard names.</span>
<span class="c"># (note: Cdc25Mm = RASGRF1)</span>
<span class="n">ras_gef_names</span> <span class="o">=</span> <span class="p">[</span><span class="s">&#39;SOS1&#39;</span><span class="p">,</span> <span class="s">&#39;SOS2&#39;</span><span class="p">,</span> <span class="s">&#39;RASGRF1&#39;</span><span class="p">,</span> <span class="s">&#39;RASGRF2&#39;</span><span class="p">]</span>
<span class="k">for</span> <span class="n">ras_gef_name</span> <span class="ow">in</span> <span class="n">ras_gef_names</span><span class="p">:</span>
    <span class="n">Monomer</span><span class="p">(</span><span class="n">ras_gef_name</span><span class="p">,</span> <span class="p">[</span><span class="s">&#39;rasgef&#39;</span><span class="p">])</span>
</pre></div>
</div>
</div>
<div class="section" id="mechanism-of-gefs">
<h2>Mechanism of GEFs<a class="headerlink" href="#mechanism-of-gefs" title="Permalink to this headline">¶</a></h2>
<p>Some key features of the mechanism:</p>
<ol class="arabic">
<li><p class="first">Ras binds GEFs in the absence of nucleotides</p>
</li>
<li><p class="first">GEF binding causes Ras to release GTP/GDP.</p>
</li>
<li><p class="first">Rebinding of nucleotides causes Ras to release the GEF.</p>
</li>
<li><p class="first">Reloading of Ras with GTP vs. GDP is not determined by the GEF, but rather
by the relative cellular concentrations of the nucleotides.</p>
<blockquote>
<div><p><a class="reference internal" href="../references.html#pmid9690470" id="id11">[PMID9690470]</a>: Biochemical studies of Ras exchange factors have shown that
the complex of Ras with these proteins is stable in the absence of
nucleotides and is dissociated by the rebinding of either GDP or GTP
(<a class="reference internal" href="../references.html#pmid9585556" id="id12">[PMID9585556]</a> <a class="reference internal" href="../references.html#pmid9690470-17" id="id13">[PMID9690470_17]</a> <a class="reference internal" href="../references.html#pmid9690470-18" id="id14">[PMID9690470_18]</a> <a class="reference internal" href="../references.html#pmid9690470-21" id="id15">[PMID9690470_21]</a>
<a class="reference internal" href="../references.html#pmid9690470-22" id="id16">[PMID9690470_22]</a>). The principal role for the exchange factor is to
facilitate nucleotide release, and it does not seem to control
significantly the preferential rebinding of GTP over GDP (<a class="reference internal" href="../references.html#pmid9585556" id="id17">[PMID9585556]</a>,
<a class="reference internal" href="../references.html#pmid9690470-22" id="id18">[PMID9690470_22]</a>, <a class="reference internal" href="../references.html#pmid7548002" id="id19">[PMID7548002]</a>).  Cellular concentrations of GTP are
10-fold higher than GDP, which results in the loading of GTP onto Ras.</p>
</div></blockquote>
</li>
</ol>
<p>The fact that GTP/GDP can displace GEFs, while GEFs can also displace GTP/GDP,
leads to a paradox that is resolved by the fact that Ras undergoes a
conformational change that retains the necessary &#8220;state&#8221;. The structural basis
of this conformational change is described as follows:</p>
<blockquote>
<div><a class="reference internal" href="../references.html#pmid9690470" id="id20">[PMID9690470]</a>: As a nucleotide-exchange factor, Sos functions under two
apparently conflicting imperatives. The interaction between Sos and Ras
must be strong enough to dislodge the tightly bound nucleotide, but the Ras
– Sos complex must also be poised for subsequent displacement by incoming
nucleotides. The structure of the Ras – Sos complex shows that Ras and Sos
meet these demands by forming a tight complex that is anchored at one end
of the nucleotide- binding site, where phosphate and magnesium are normally
bound. The interface between Sos and Ras is mainly hydrophilic, suggesting
a ready unzippering through water-mediated displacements of the
coordinating side chains. The main interacting elements of Sos avoid direct
occlusion of the nucleotide-binding site, except the region where the
terminal phosphate groups and the magnesium ion are bound. This feature
allows incoming nucleotides to reverse the process by competing for the
groups that ligate the phosphate and metal ion.</div></blockquote>
<p>This conformational state change has been analyzed kinetically and identified
as the process of the nucleotide being first loosely and then tightly bound,
(see also <a class="reference internal" href="ras_gtpase.html#ras-gtpase"><span>Ras proteins are GTPases</span></a>):</p>
<blockquote>
<div><a class="reference internal" href="../references.html#pmid9690470" id="id21">[PMID9690470]</a>: Kinetic analysis of nucleotide association shows that the
reaction proceeds by the formation of a ternary complex of a loosely bound
nucleotide and Ras – Cdc25Mm followed by conversion to a form in which the
nucleotide is tightly bound to Ras <a class="reference internal" href="../references.html#pmid9585556" id="id22">[PMID9585556]</a>. In light of the
structure of the Ras–Sos complex, the first step can be interpreted as the
interaction of the base and the ribose of the nucleotide with the part of
the Ras binding site that is not occluded by Sos. The second step would
involve a conformational change in the Switch 2 segment and release of
Switch 1, resulting in the restructuring of a competent binding site for
phosphate and magnesium, and the subsequent dissociation of Sos.</div></blockquote>
<p>The kinetic analysis described in <a class="reference internal" href="../references.html#pmid9585556" id="id23">[PMID9585556]</a> resulted in the following reaction scheme for the interactions between Ras, GTP/GDP, and GEFs:</p>
<a class="reference internal image-reference" href="../../_images/9585556_rasgef_cycle.png"><img alt="../../_images/9585556_rasgef_cycle.png" src="../../_images/9585556_rasgef_cycle.png" style="width: 600px;" /></a>
<p>Note that the upper equilibria for Ras-nucleotide binding, K1a and K1b, were
implemented in the section <a class="reference internal" href="ras_gtpase.html#ras-gtpase"><span>Ras proteins are GTPases</span></a>, along with corresponding rates.
Here we implement only the equilibria involving GEFs: K2, K3, K4a and K4b.</p>
<div class="highlight-python"><div class="highlight"><pre><span class="k">def</span> <span class="nf">ras_gef_exchange_cycle</span><span class="p">(</span><span class="n">ras</span><span class="p">,</span> <span class="n">rasgef</span><span class="p">,</span> <span class="n">gxp</span><span class="p">,</span>
                           <span class="n">k2_list</span><span class="p">,</span> <span class="n">k3_list</span><span class="p">,</span> <span class="n">k4a_list</span><span class="p">,</span> <span class="n">k4b_list</span><span class="p">):</span>
    <span class="c"># Alias for Ras bound to GXP</span>
    <span class="n">rasgxp</span> <span class="o">=</span> <span class="n">ras</span><span class="p">(</span><span class="n">gef</span><span class="o">=</span><span class="bp">None</span><span class="p">,</span> <span class="n">gtp</span><span class="o">=</span><span class="mi">99</span><span class="p">)</span> <span class="o">%</span> <span class="n">gxp</span><span class="p">(</span><span class="n">p</span><span class="o">=</span><span class="mi">99</span><span class="p">)</span>

    <span class="c"># Binding of RasGEF to nucleotide-free Ras (K2)</span>
    <span class="n">bind</span><span class="p">(</span><span class="n">ras</span><span class="p">(</span><span class="n">gtp</span><span class="o">=</span><span class="bp">None</span><span class="p">,</span> <span class="n">s1s2</span><span class="o">=</span><span class="s">&#39;closed&#39;</span><span class="p">),</span> <span class="s">&#39;gef&#39;</span><span class="p">,</span> <span class="n">rasgef</span><span class="p">(),</span> <span class="s">&#39;rasgef&#39;</span><span class="p">,</span> <span class="n">k2_list</span><span class="p">)</span>

    <span class="c"># Binding of RasGEF to RasGXP (K3)</span>
    <span class="n">bind</span><span class="p">(</span><span class="n">rasgxp</span><span class="p">(</span><span class="n">s1s2</span><span class="o">=</span><span class="s">&#39;open&#39;</span><span class="p">),</span> <span class="s">&#39;gef&#39;</span><span class="p">,</span> <span class="n">rasgef</span><span class="p">(),</span> <span class="s">&#39;rasgef&#39;</span><span class="p">,</span> <span class="n">k3_list</span><span class="p">)</span>

    <span class="c"># Binding of GXP to Ras/RasGEF complex</span>
    <span class="n">bind</span><span class="p">(</span><span class="n">ras</span><span class="p">(</span><span class="n">s1s2</span><span class="o">=</span><span class="s">&#39;closed&#39;</span><span class="p">,</span> <span class="n">gef</span><span class="o">=</span><span class="mi">1</span><span class="p">)</span> <span class="o">%</span> <span class="n">rasgef</span><span class="p">(</span><span class="n">rasgef</span><span class="o">=</span><span class="mi">1</span><span class="p">),</span> <span class="s">&#39;gtp&#39;</span><span class="p">,</span>
         <span class="n">gxp</span><span class="p">(),</span> <span class="s">&#39;p&#39;</span><span class="p">,</span> <span class="n">k4a_list</span><span class="p">)</span>

    <span class="c"># Isomerization of Ras-RasGEF-GXP from loose to tight</span>
    <span class="n">equilibrate</span><span class="p">(</span><span class="n">rasgxp</span><span class="p">(</span><span class="n">gef</span><span class="o">=</span><span class="mi">1</span><span class="p">,</span> <span class="n">s1s2</span><span class="o">=</span><span class="s">&#39;closed&#39;</span><span class="p">)</span> <span class="o">%</span> <span class="n">rasgef</span><span class="p">(</span><span class="n">rasgef</span><span class="o">=</span><span class="mi">1</span><span class="p">),</span>
                <span class="n">rasgxp</span><span class="p">(</span><span class="n">gef</span><span class="o">=</span><span class="mi">1</span><span class="p">,</span> <span class="n">s1s2</span><span class="o">=</span><span class="s">&#39;open&#39;</span><span class="p">)</span> <span class="o">%</span> <span class="n">rasgef</span><span class="p">(</span><span class="n">rasgef</span><span class="o">=</span><span class="mi">1</span><span class="p">),</span> <span class="n">k4b_list</span><span class="p">)</span>
</pre></div>
</div>
</div>
<div class="section" id="rates-of-gef-activation">
<h2>Rates of GEF activation<a class="headerlink" href="#rates-of-gef-activation" title="Permalink to this headline">¶</a></h2>
<div class="highlight-python"><div class="highlight"><pre><span class="c"># Binding of RasGEF to nucleotide-free Ras</span>
<span class="n">kf2</span> <span class="o">=</span> <span class="mf">0.33e6</span>        <span class="c"># M^-1 s^-1</span>
<span class="n">kr2</span> <span class="o">=</span> <span class="mf">1e-3</span>          <span class="c"># s^-1</span>

<span class="c"># Binding of RasGEF to RasGXP</span>
<span class="n">KD3</span> <span class="o">=</span> <span class="mf">0.6e-3</span>        <span class="c"># M</span>
<span class="n">kf3</span> <span class="o">=</span> <span class="mf">3.4e4</span>         <span class="c"># M^-1 s^-1 (lower limit)</span>
<span class="n">kr3</span> <span class="o">=</span> <span class="n">KD3</span> <span class="o">*</span> <span class="n">kf3</span>     <span class="c"># s^-1</span>

<span class="c"># Binding of GXP to Ras/RasGEF complex</span>
<span class="n">KD4a</span> <span class="o">=</span> <span class="mf">8.6e-6</span>       <span class="c"># M</span>
<span class="n">kf4a</span> <span class="o">=</span> <span class="mf">1e7</span>          <span class="c"># M^-1 s^-1</span>
<span class="n">kr4a</span> <span class="o">=</span> <span class="n">KD4a</span> <span class="o">*</span> <span class="n">kf4a</span>  <span class="c"># s^-1</span>
</pre></div>
</div>
<p># = kf1a, i.e., on rate is insensitive to presence of GEF</p>
<div class="highlight-python"><div class="highlight"><pre><span class="c"># Isomerization of Ras-RasGEF-GXP from loose to tight</span>
<span class="n">kf4b</span> <span class="o">=</span> <span class="mf">20.4</span>         <span class="c"># s^-1</span>
<span class="n">kr4b</span> <span class="o">=</span> <span class="mf">3.9</span>          <span class="c"># s^-1</span>
</pre></div>
</div>
<p>The following study used purified HRAS and mouse RASGRF1:</p>
<blockquote>
<div><p><a class="reference internal" href="../references.html#pmid9690470" id="id24">[PMID9690470]</a>: The mechanism of nucleotide release by the catalytic domain
of murine Cdc25 (Cdc25Mm) has been investigated recently using
fluorescently labelled nucleotides <a class="reference internal" href="../references.html#pmid9585556" id="id25">[PMID9585556]</a>.  The affinity of Cdc25Mm
for nucleotide-free Ras (Kd = 4.6 nM) is found to be several orders of
magnitude higher than that for nucleotide-bound Ras, and the maximal
acceleration by Cdc25Mm of the rate of dissociation of nucleotide is more
than 10^5.</p>
<p><a class="reference internal" href="../references.html#pmid9585556" id="id26">[PMID9585556]</a>: The best fit of our data resulted in similar quantum yields
and a value of 4.6 nM for KD2 (NOTE: Kd between nucleotide-free H-Ras and
RasGRF1). A variation in the value for KD2 of approximately 2-fold resulted
in fits of comparable quality.</p>
</div></blockquote>
<p>The activity of GEF (RASGRF1 in this case) does not depend on whether Ras
(HRAS) is loaded with GTP or GDP.</p>
<blockquote>
<div><p><a class="reference internal" href="../references.html#pmid9585556" id="id27">[PMID9585556]</a>: However, since the intrinsic dissociation rate of Ras for
GTP (1 × 10-5 s-1) is 2-fold lower than that for GDP (2 × 10-5 s-1), the
stimulatory action of Cdc25Mm285 is practically independent of the nature
of the bound nucleotide.</p>
<p><a class="reference internal" href="../references.html#pmid9585556" id="id28">[PMID9585556]</a>: Although we did not reach complete saturation at 600 μM
Ras‚nucleotide, the data could be fitted to obtain a maximal rate of
3′mdGDP release from Ras of 3.9 s-1 and an apparent Km value of 386 μM.
Since the intrinsic dissociation rate of 3′mdGDP is 2 × 10-5 s-1 (Table 1),
the acceleration of GDP dissociation from Ras by this GEF is approximately
2 × 10^5-fold. An apparent Km of approximately 300 μM was obtained
for the triphosphate-bound form of Ras, confirming that there is no
pronounced specificity toward the nature of the Ras-bound nucleotide (data
not shown).</p>
</div></blockquote>
<div class="admonition warning">
<p class="first admonition-title">Warning</p>
<p>GEF binding to GTP bound Ras?</p>
<p class="last">Can GEFs bind to Ras and cause ejection of nucleotide before the GTP/GDP
conversion is complete? Moreover, if GEF binds to Ras-GTP, can the
hydrolysis to GDP proceed while GEF is bound?</p>
</div>
<p>Instantiate the RasGEF cycle for HRAS and RASGRF1:</p>
<div class="highlight-python"><div class="highlight"><pre><span class="c">#ras_gef_exchange_cycle(HRAS, RASGRF1, GTP, GDP)</span>
</pre></div>
</div>
<p><a class="reference internal" href="../references.html#pmid9585556" id="id29">[PMID9585556]</a>: Therefore, we tested the nucleotide specificity of the
interaction of Cdc25Mm285 (CdcMm285 is the fragment of CdcMm/RasGRF1 containing
the RasGEF domain) with Ras. Figure 1 shows the release of Ras-bound 3′mdGDP or
3′mdGTP (4 μM), in the presence of an excess of unlabeled nucleotide and in the
presence or absence of 1 μM Cdc25Mm285. The Cdc25Mm285-stimulated dissociation
rate of Ras-3′mdGDP is approximately twice that of Ras-3′mdGTP, with values of
0.0098 and 0.0046 s-1, respectively.  However, since the intrinsic dissociation
rate of Ras for GTP (1 × 10-5 s-1) is 2-fold lower than that for GDP (2 × 10-5
s-1), the stimulatory action of Cdc25Mm285 is practically independent of the
nature of the bound nucleotide. The difference in stimulated dissociation rates
is somewhat smaller than the results of Jacquet et al. (16) but is similar to
the results with the yeast proteins CDC25 and RAS2 obtained by Haney and Broach
(28).</p>
<p><a class="reference internal" href="../references.html#pmid9690470" id="id30">[PMID9690470]</a>: The overall shape of the catalytic domain of Sos is that of an
oblong bowl (Fig. 2), with Ras bound at the centre of the bowl. The regions of
Ras that interact most closely with Sos include the phosphate-binding P-loop
(residues 10 – 17) and surrounding segments (including strand 􏰧1 and helix 􏰦1),
the Switch 1 region (defined here as residues 25–40) and the Switch 2 region
(defined here as residues 57 – 75). Additional interactions are seen with helix
3 (residues 95–105; Fig. 3a, b). The interface between Ras and Sos is primarily
hydrophilic and very extensive, with 3,600 A^2 of surface area buried in the
complex.</p>
<p><a class="reference internal" href="../references.html#pmid9690470" id="id31">[PMID9690470]</a>: The most obvious effect of Sos binding to Ras is the opening of
the nucleotide binding site as a result of the displacement of Switch 1 of Ras
by the insertion of the helical hairpin formed by aH and aI of Sos (Fig. 5)</p>
<p>Switch 1 and Switch 2 are the only regions of Ras in which structural changes
are directly induced by Sos.</p>
<p>The change in the Switch 1 region of Ras when bound to Sos is drastic...Switch
1 is completely removed from the nucleotide-binding site.</p>
<p>One important aspect of the insertion of the helical hairpin of Sos into the
Switch 1 region is that it does not result in a significant occlusion of the
guanine and ribose binding sites (Fig. 5d). Instead, this structural distortion
breaks the network of direct and water-mediated interactions between Switch 1
and the nucleotide. For example, in the nucleotide-bound forms of Ras, Phe 28
interacts with the guanine base through a perpendicular aromatic – aromatic
interaction (Fig. 5a). Mutation of Phe28 to leucine results in a significant
increase in the intrinsic rate of dissociation of nucleotide from Ras18. In the
Sos complex, the Calpha of Phe 28 moves 9.6 A and the side chain no longer
interacts with the nucleotide-binding site (Fig. 5b).</p>
<p>The Switch 2 region of Ras makes important interactions with GTP and not with
GDP (19,46). Nevertheless, structural changes that are induced in Switch 2 by
Sos result in the exclusion of both GDP and GTP, because they affect magnesium
binding as well as the conformation of Lys 16 in the P-loop, a crucial
phosphate ligand.</p>
</div>
<div class="section" id="specificity-of-rasgrf1-for-ras-isoforms">
<h2>Specificity of RASGRF1 for Ras isoforms<a class="headerlink" href="#specificity-of-rasgrf1-for-ras-isoforms" title="Permalink to this headline">¶</a></h2>
<p><a class="reference internal" href="../references.html#pmid9585556" id="id32">[PMID9585556]</a>: Three mammalian isoforms of Ras, H-, K-, and N-Ras, have been
identified which are highly conserved intheirprimarysequence.
Thesignificanceofhavingmore than one isoform is not understood at present,
although the isoforms may have different functions in different tissues, since
certain types of tumors have a preference for a particular activated Ras gene,
such as K-Ras for lung, colon and pancreas cancers and N-Ras for myeloid
leukemias (25). To see whether Cdc25Mm285 acts differently on the three
isoforms, we tested the GEF activity of Cdc25Mm285 on these proteins. As
summarized in Table 1, Cdc25Mm285 is active on all isoforms, being somewhat
more active on N-Ras, in accordance with the results of Leonardsen et al. (26).</p>
<script>
    window.setTimeout(function() {
    $('div.highlight-python pre > span.c:last-child').each(
        function () {
            if ($(this).text() == '#') {
                $(this.nextSibling).detach();
                $(this).detach();
            }
        }
    );
    }, 1000);
</script></div>
</div>


           </div>
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="ras_gaps.html" class="btn btn-neutral float-right" title="Ras regulation by RasGAPs" accesskey="n">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="ras_gtpase.html" class="btn btn-neutral" title="Ras proteins are GTPases" accesskey="p"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2015, Sorger Lab.

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  

    <script type="text/javascript">
        var DOCUMENTATION_OPTIONS = {
            URL_ROOT:'../../',
            VERSION:'0.1',
            COLLAPSE_INDEX:false,
            FILE_SUFFIX:'.html',
            HAS_SOURCE:  true
        };
    </script>
      <script type="text/javascript" src="../../_static/jquery.js"></script>
      <script type="text/javascript" src="../../_static/underscore.js"></script>
      <script type="text/javascript" src="../../_static/doctools.js"></script>
      <script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>

  

  
  
    <script type="text/javascript" src="../../_static/js/theme.js"></script>
  

  
  
  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.StickyNav.enable();
      });
  </script>
   

</body>
</html>